templates/company/button_company.html.twig

```diff
@@ -8,6 +8,7 @@
             <div class="modalSpace"></div>
             <div class="multiCol">
                 {% for company in job.companies %}
+                    {% if company.accepted == 1 %}
```

Comment:
test company.accepted is true plutot que ==1

Reply:
et tu peux tout mettre sur la meme ligne en Twig pour economiser le block if/endif
                 {% for company in job.companies if company.accepted is true %}